### PR TITLE
Update discussion labels in project portfolio page

### PR DESCRIPTION
**Pull request by w2vgd**

### CODE REVIEW

**Review by candyhy — APPROVED** [review]
LGTM

File: docs/team/w2vgd.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Given below are my contributions to the project:
     * Highlights: Each `Property` may contain several optional attributes, such as remarks, tags and client's information. Each attribute also has nicely formatted output to be shown to the user in the GUI.
   
 
-* **Enhancement**: Added the ability to add properties and appointments through the two different `add` commands.
+* **Enhancement**: Added the functionality to add properties and appointments through the two different `add` commands.
```
Comment (dvdweien) [inline docs/team/w2vgd.md:22]: imo ability reads smoother

File: docs/team/w2vgd.md [hunk 2]
```diff
@@ -52,32 +52,21 @@ Given below are my contributions to the project:
...
-
-* Provided help through forum discussions (examples: [1](https://github.com/nus-cs2103-AY2021S2/forum/issues/5), [2](https://github.com/nus-cs2103-AY2021S2/forum/issues/30), [3](https://github.com/nus-cs2103-AY2021S2/forum/issues/53))
-
-* Helped fix a bug in the AB3 Logic class diagram that is adopted by other class amtes (examples: [1](https://github.com/nus-cs2103-AY2021S2/forum/issues/257))
+* PRs reviewed (with non-trivial review comments): ([\#132](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/132), [\#202](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/202), [\#191](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/191), [\#195](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/195), [\#84](https://github.com/AY2021S2-CS2103T-T13-4/tp/pull/84))
+* Initiated forum discussions tagged with `must-see` labels (discussion examples: [\#224](https://github.com/nus-cs2103-AY2021S2/forum/issues/224), [\#257](https://github.com/nus-cs2103-AY2021S2/forum/issues/257), [\#276](https://github.com/nus-cs2103-AY2021S2/forum/issues/276))
```
Comment (dvdweien) [inline docs/team/w2vgd.md:69]: this one op sia

**Review by dvdweien — COMMENTED** [review]
just a comment, will merge if u dont agree

**Pull request merged**